### ISSUE TITLE
Add ObexFTP support

### ISSUE THREAD

**Issue by OffifialMITX** [arpitkh96/AmazeFileManager#404]
At first, I've been using this file manager for a while and this is the best one I could find on F-Droid - great work!

The only feature I am really missing is to connect to a bluetooth device and it would be very cool if it was possible.

**Comment by vishal0071** [comment]
You can share files using bluetooth, how else would you like to connect?

**Comment by OffifialMITX** [comment]
Of course, it works fine while sending one or two files but if you want to backup data from your old phone or one with a broken screen or just using bluetooth often and not needing to look at both devices at the same time.

**Comment by KOLANICH** [comment]
:+1:
